packages/graph/hash_graph/lib/graph/src/ontology/entity_type.rs

```diff
@@ -79,8 +81,8 @@ pub struct EntityTypeQueryPathVisitor {
 impl EntityTypeQueryPathVisitor {
     pub const EXPECTING: &'static str = "one of `ownedById`, `createdById`, `updatedById`, \
                                          `removedById`, `baseUri`, `versionedUri`, `version`, \
-                                         `title, `description`, `default`, `examples`, \
-                                         `properties`, `required`, `links`, or `requiredLinks`";
+                                         `title`, `description`, `default`, `examples`, \
+                                         `properties`, `required`, `links`, `requiredLinks`";
```

Comment:
The removal of the `or` at the end is intentional, right? Is it just to make it easier to add/change or is there another reason?